<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dlg_set_color</class>
 <widget class="QDialog" name="Dlg_set_color">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>颜色设置</string>
  </property>
  <widget class="QPushButton" name="m_btn_cancel">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>510</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_wav_ibp1">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>360</y>
     <width>85</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>60</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>心率</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>15</y>
     <width>3</width>
     <height>30</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_co2">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>300</y>
     <width>85</width>
     <height>30</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_wav_ecg7">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>270</y>
     <width>85</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_wav_ecg5">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>210</y>
     <width>85</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_nibp">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>180</y>
     <width>85</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_beat_rate">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>60</y>
     <width>85</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>240</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ECG6</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>120</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ECG2</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_ibp2">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>270</y>
     <width>85</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_spo2">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>90</y>
     <width>85</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>240</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>IBP1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>335</x>
     <y>20</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>颜色</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>270</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>IBP2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>210</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ECG5</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_wav_pulse">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>300</y>
     <width>85</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>60</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>脑电波</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_temp">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>150</y>
     <width>85</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>330</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>呼吸波</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>150</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ECG3</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_spire_rate">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>120</y>
     <width>85</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_wav_ecg3">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>150</y>
     <width>85</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>150</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>体温</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_ibp1">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>240</y>
     <width>85</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>210</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>麻醉深度</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>300</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>CO2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>120</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>呼吸率</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>90</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ECG1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>20</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>参数</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_wav_co2">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>420</y>
     <width>85</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_wav_spr">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>330</y>
     <width>85</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>270</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ECG7</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>420</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>CO2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>300</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>脉搏波</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_wav_mashen">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>450</y>
     <width>85</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_27">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>20</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>颜色</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_25">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>20</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>波形</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_26">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>450</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>麻醉深度</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_wav_ecg2">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>120</y>
     <width>85</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>390</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>IBP2</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>215</x>
     <y>15</y>
     <width>3</width>
     <height>450</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>90</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>血氧</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btn_ok">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>510</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_wav_ecg6">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>240</y>
     <width>85</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_mashen">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>210</y>
     <width>85</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>295</x>
     <y>15</y>
     <width>3</width>
     <height>30</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>180</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>无创血压</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_wav_ecg4">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>180</y>
     <width>85</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_wav_eeg">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>60</y>
     <width>85</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>360</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>IBP1</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_wav_ibp2">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>390</y>
     <width>85</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_wav_ecg1">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>90</y>
     <width>85</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>180</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ECG4</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
